src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -35,22 +35,26 @@ function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
         {
             title: translate('workspace.qbo.advancedConfig.autoSync'),
             subtitle: translate('workspace.qbo.advancedConfig.autoSyncDescription'),
-            isActive: Boolean(autoSync),
-            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, !autoSync),
+            isActive: Boolean(autoSync?.enabled),
+            onToggle: () =>
+                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, {
+                    jobID: autoSync?.jobID ?? '',
+                    enabled: !autoSync?.enabled,
+                }),
```

Comment:
We can't send an object to the server. This probably need some work. I'm guessing we only need to send a boolean value here but this function is linked with Onyx types and the type in onyx is an object.